lib/helpers.py

```diff
@@ -70,7 +70,14 @@ def current_solution_or_project_json_folder(view):
             solution_file_path = os.path.abspath(solution_file_path)
             return solution_file_path
     else:
-        parentpath = sublime.active_window().folders()[0] #assume parent folder is opened that contains all project folders eg/Web,ClassLib,Tests
+        try:
+            parentpath = sublime.active_window().folders()[0] #assume parent folder is opened that contains all project folders eg/Web,ClassLib,Tests
+        except Exception:
+            try:
+                parentpath = os.path.dirname(sublime.active_window().active_view().file_name())
+            except Exception:

```

Comment:
If you create a new file that is unsaved and specify a syntax for it in sublime `View - Syntax -C#` we could use that with omnisharp.